src/libostree/ostree-repo.h

```diff
@@ -462,9 +462,11 @@ gboolean      ostree_repo_resolve_rev (OstreeRepo  *self,
 /**
  * OstreeRepoResolveRevExtFlags:
  * @OSTREE_REPO_RESOLVE_REV_EXT_NONE: No flags.
+ * @OSTREE_REPO_RESOLVE_REV_EXT_LOCAL_ONLY: Exclude remote and mirrored refs. Since: 2019.2
  */
 typedef enum {
   OSTREE_REPO_RESOLVE_REV_EXT_NONE = 0,
+  OSTREE_REPO_RESOLVE_REV_EXT_LOCAL_ONLY = (1 << 0),
```

Comment:
Would it make sense to expose two separate flags, `EXCLUDE_MIRRORS` and `EXCLUDE_REMOTES`, like we have for `OstreeRepoListRefsExtFlags`? Might make things a bit more consistent.

Reply:
I get that the symmetry would be nice, but I don't think it's a distinction we need to make. Because remote and mirrored refs are both remote and both mirrored (in the sense that we serve both over LAN/USB). I'm trying to move Flatpak to use `refs/mirrors/` more, at which point the only distinction will be the on-disk location.

I think the only place the distinction between EXCLUDE_REMOTES and EXCLUDE_MIRRORS is needed for ostree_repo_list_refs() is in the implementation of ostree_repo_traverse_reachable_refs(), because otherwise you'd check the same refs twice.

Reply:
Shrug. Maybe a second opinion from @cgwalters would be good.

Reply:
No strong opinion here either.

Reply:
Okay, we'll go with `LOCAL_ONLY` then